No HUD is displayed with unchained economy on PF1 v11

Hey, first off thanks for this amazing module!

When I click on an actor, no HUD elements are displayed. I think it's related to unchained action economy, I'll explain my finding further below.

Here are my enabled modules:
- Argon - Combat HUD (CORE)
- Argon - Combat HUD (PF1)
- Better rolls formulas
- Chat Commander
- Color Picker
- lib - Color Settings
- libWrapper
- Tagger

FoundryVTT Version is 12 Stable, Build 331.
Pathfinder 1e version is 11.1.

How to reproduce:
- Enable unchained action economy.
- (optional?) Enable "Always on" on Argon core settings
- Click on an actor with items with actions (in my case, spells) and enable Argon HUD if need be.
- No HUD elements are displayed.

So, I tried to take a look. I've observed two errors in the console. 

The first one occurs when I click on an actor:
```
foundry.js:655 Error: An error occurred while rendering CoreHUD 25. Cannot read properties of undefined (reading 'type')
[Detected 2 packages: enhancedcombathud-pf1(0.11.2), enhancedcombathud(3.1.0)]
    at Hooks.onError (foundry.js:654:24)
    at 🎁Hooks.onError#0 (libWrapper-wrapper.js:188:54)
    at foundry.js:5795:13Caused by: TypeError: Cannot read properties of undefined (reading 'type')
[Detected 2 packages: enhancedcombathud-pf1(0.11.2), enhancedcombathud(3.1.0)]
    at buttonPanelButton.mjs:298:57
    at Array.filter (<anonymous>)
    at get validItems (buttonPanelButton.mjs:275:32)
    at get isValid (buttonPanelButton.mjs:315:70)
    at panels.mjs:398:52
    at Array.filter (<anonymous>)
    at Pathfinder1eUnchainedActionPanel._getButtons (panels.mjs:398:28)
    at CoreHUD._renderInner (hud.js:273:49)
    at async CoreHUD._render (foundry.js:5844:19)
```

It seems the issue is caused by this code, in `buttonPaneButton.mjs`: 
```javascript
get validItems() {
    const itemsOfType = this.actor.items.filter(item => item.type === "spell" && item.system.spellbook === this.spellbookId);
    return itemsOfType.filter(item => {
        if (!item.actions) {
            return false;
        }

        if (item.type === "spell") {
            if (!item.canUse) {
                return false;
            }

            if (item.spellbook.spellPreparationMode === "prepared" && !item.useSpellPoints()) {
                if (item.system.preparation.max === 0) {
                    return false;
                }
            } else {
                if (!item.system.preparation.value) {
                    return false;
                }
            }
        }

        for (let action of item.actions) {
            if (this.isUnchained) {
                if (action.activation.unchained.type === this.actionType) {
                    return true;
                }
            } else {
                if (action.activation.type === this.actionType
                    && action.activation.cost === 1) {
                    return true;
                }
            }
        }

        return false;
    });
}
```

Even though `this.isUnchained` is `true`, `action.activation.unchained` key is `undefined`.

This is something I also see in the console, when I display actor's information: 

<img width="446" alt="Image" src="https://github.com/user-attachments/assets/91ef3cd6-350a-419c-a604-9438890417cc" />

Oddly enough, this isn't the case when I use Koboldworks - Data Inspector? (I think both screenshots are about the same item)
<img width="238" alt="Image" src="https://github.com/user-attachments/assets/03e8f215-f78b-4be4-955d-fc04693b5c5b" />

Anyway, the second error is when I close an actor sheet, another error is displayed, not sure if it's related to the first:
```
foundry.js:655 Error: Error thrown in hooked function 'bound _onUpdateActor' for hook 'updateActor'. Cannot set properties of null (setting 'innerHTML')
[Detected 2 packages: enhancedcombathud(3.1.0), system:pf1(11.1)]
    at Hooks.onError (foundry.js:654:24)
    at 🎁Hooks.onError#0 (libWrapper-wrapper.js:188:54)
    at #call (foundry.js:636:36)
    at Hooks.callAll (foundry.js:589:17)
    at foundry-esm.js:56584:17
    at foundry-esm.js:56590:43
    at Array.map (<anonymous>)
    at #handleUpdateDocuments (foundry-esm.js:56590:33)
    at async ActorCharacterPF.updateDocuments (foundry-esm.js:11473:23)
    at async ActorCharacterPF.update (foundry-esm.js:11584:23)
    at async ActorSheetPFCharacter._onSubmit (foundry.js:6665:7)
    at async ActorSheetPFCharacter._onSubmit (actor-sheet.mjs:4148:52)
    at async ActorSheetPFCharacter.submit (foundry.js:6936:5)
    at async ActorSheetPFCharacter.close (foundry.js:6909:19)
    at async ActorSheetPFCharacter.close (foundry.js:7095:5)Caused by: TypeError: Cannot set properties of null (setting 'innerHTML')
[Detected 2 packages: enhancedcombathud(3.1.0), system:pf1(11.1)]
    at PassTurnPanel.updateActionUse (actionPanel.js:48:34)
    at hud.js:177:63
    at Array.forEach (<anonymous>)
    at CoreHUD._onUpdateActor (hud.js:177:30)
    at #call (foundry.js:632:20)
    at Hooks.callAll (foundry.js:589:17)
    at foundry-esm.js:56584:17
    at foundry-esm.js:56590:43
    at Array.map (<anonymous>)
    at #handleUpdateDocuments (foundry-esm.js:56590:33)
    at async ActorCharacterPF.updateDocuments (foundry-esm.js:11473:23)
    at async ActorCharacterPF.update (foundry-esm.js:11584:23)
    at async ActorSheetPFCharacter._onSubmit (foundry.js:6665:7)
    at async ActorSheetPFCharacter._onSubmit (actor-sheet.mjs:4148:52)
    at async ActorSheetPFCharacter.submit (foundry.js:6936:5)
    at async ActorSheetPFCharacter.close (foundry.js:6909:19)
    at async ActorSheetPFCharacter.close (foundry.js:7095:5)
```

Here is the actor I've experimented with, but the issue seems to appear on any complex actor.
[fvtt-Actor-miroshi,-la-brise-ecarlate-IpaW3zsW3DNGht69.json](https://github.com/user-attachments/files/18775393/fvtt-Actor-miroshi.-la-brise-ecarlate-IpaW3zsW3DNGht69.json)

--- a/CHANGELOG
+++ b/CHANGELOG
@@ -1,3 +1,11 @@
+## 0.12.0
+- Add Chinese Translation (Huge Thanks to [@NaricaNardica](https://github.com/NaricaNardica))
+- Add French Translation (Huge Thanks to [@guillaume-gc](https://github.com/guillaume-gc))
+- Fix unchained action economy crashing combat hud
+- Fix equipment tooltips showing broken data
+- No longer exclude actions with an activation cost larger than 1 action (mostly relevant for UC action economy)
+- Fix unchained action economy cost not being properly deducted
+
 ## 0.11.2
 - Fix item tooltips breaking when no material data is available
 - Fix additional deprecation warnings

--- a/README.md
+++ b/README.md
@@ -8,7 +8,7 @@ This module currently supports the following languages:
 - English
 - German (Deutsch)
 - French (Huge Thanks to [@guillaume-gc](https://github.com/guillaume-gc))
-- Chinese (Hue Thanks to [@NaricaNardica](https://github.com/NaricaNardica))
+- Chinese (Huge Thanks to [@NaricaNardica](https://github.com/NaricaNardica))
 
 Should you wish to contribute further languages, please feel free to open a pull request or contact me directly.
 

--- a/module.json
+++ b/module.json
@@ -1,7 +1,7 @@
 {
   "id": "enhancedcombathud-pf1",
   "title": "Argon - Combat HUD (PF1)",
-  "version": "0.11.2",
+  "version": "0.12.0",
   "compatibility": {
     "minimum": "12",
     "verified": "12"

--- a/modules/argon/buttons/buttonPanelButton.mjs
+++ b/modules/argon/buttons/buttonPanelButton.mjs
@@ -136,28 +136,14 @@ export function buttonPanelItemButton(ARGON) {
         get validItems() {
             const itemsOfType = this.actor.items.filter(item => item.type === this.type);
             return itemsOfType.filter(item => {
-                if (!item.actions) {
+                if (
+                    !item.actions?.size
+                    || (item.isCharged && !item.charges)
+                ) {
                     return false;
                 }
 
-                if (item.isCharged && !item.charges) {
-                    return false;
-                }
-
-                for (let action of item.actions) {
-                    if (this.isUnchained) {
-                        if (action.activation.unchained.type === this.actionType) {
-                            return true;
-                        }
-                    } else {
-                        if (action.activation.type === this.actionType
-                            && action.activation.cost === 1) {
-                            return true;
-                        }
-                    }
-                }
-
-                return false;
+                return item.actions.some(action => action.activation.type === this.actionType && action.activation.cost <= this.parent.maxActions);
             });
         }
 
@@ -173,8 +159,10 @@ export function buttonPanelItemButton(ARGON) {
             switch (this.type) {
                 case "consumable":
                     return `modules/${ModuleName}/icons/vial.svg`;
+
                 case "equipment":
                     return `modules/${ModuleName}/icons/gem-chain.svg`;
+
                 case "feat":
                     return "modules/enhancedcombathud/icons/svg/mighty-force.svg";
             }
@@ -293,20 +281,7 @@ export function spellbookButtonPanelActionButton(ARGON) {
                     }
                 }
 
-                for (let action of item.actions) {
-                    if (this.isUnchained) {
-                        if (action.activation.unchained.type === this.actionType) {
-                            return true;
-                        }
-                    } else {
-                        if (action.activation.type === this.actionType
-                            && action.activation.cost === 1) {
-                            return true;
-                        }
-                    }
-                }
-
-                return false;
+                return item.actions.some(action => action.activation.type === this.actionType && action.activation.cost <= this.parent.maxActions);
             });
         }
 
@@ -381,7 +356,7 @@ export function spellButtonPanelActionButton(ARGON) {
                 if (item.type !== "spell") continue;
 
                 if (!item.canUse) {
-                   continue;
+                    continue;
                 }
 
                 if (item.spellbook.spellPreparationMode === "prepared" && !item.useSpellPoints()) {
@@ -396,19 +371,10 @@ export function spellButtonPanelActionButton(ARGON) {
 
                 if (!usedSpellbooks.includes(item.system.spellbook)) continue;
 
-                if (!item.actions) continue;
+                if (!item.actions?.size) continue;
 
-                for (let action of item.actions) {
-                    if (this.isUnchained) {
-                        if (action.activation.unchained.type === this.actionType) {
-                            return true;
-                        }
-                    } else {
-                        if (action.activation.type === this.actionType
-                            && action.activation.cost === 1) {
-                            return true;
-                        }
-                    }
+                if(item.actions.some(action => action.activation.type === this.actionType && action.activation.cost <= this.parent.maxActions)) {
+                    return true;
                 }
             }
 

--- a/modules/argon/buttons/itemButton.mjs
+++ b/modules/argon/buttons/itemButton.mjs
@@ -37,7 +37,7 @@ export function itemButton(ARGON) {
         }
 
         get actionCost() {
-            return this.isUnchained ? this.item.defaultAction?.activation.unchained.cost : 1;
+            return this.isUnchained ? this.item.defaultAction?.activation.cost : 1;
         }
 
         get isValid() {
@@ -175,15 +175,15 @@ export function itemButton(ARGON) {
 
                 case "equipment":
                     if (item.system.subType === "wondrous") {
-                        subtitle = item.system.subType ? pf1.config.equipmentSlots[item.system.slot] : null;
+                        subtitle = item.system.subType ? pf1.config.equipmentSlots.wondrous[item.system.slot] : null;
                     } else {
-                        subtitle = item.system.subType ? pf1.config.equipmentTypes[item.system.subType] : null;
+                        subtitle = item.system.subType ? pf1.config.equipmentTypes[item.system.subType]._label : null;
                     }
 
                     if (item.system.armor.value) {
                         details.push({
                             label: game.i18n.localize("PF1.ACNormal"),
-                            value: ('+' + item.system.armor.value).replace('+-', '-')
+                            value: ('+' + (item.system.armor.value + item.system.armor.enh)).replace('+-', '-')
                         })
                         details.push({
                             label: game.i18n.localize("PF1.MaxDexShort"),

